doc(release) Release notes oss 25.10 juil.26 (#5513)
centreon/centreon-documentation: versioned_docs/version-25.10/releases/centreon-os.mdx

| **Centreon AWIE** | [25.10.8 (July 22, 2026)](#centreon-awie-25108)<br/>[25.10.7 (June 29, 2026)](#centreon-awie-25107)<br/>[25.10.6 (April 28, 2026)](#centreon-awie-25106)<br/>[25.10.5 (Febuary 25, 2026)](#centreon-awie-25105)<br/>[25.10.3 (January 26, 2026)](#centreon-awie-25103)<br/>[25.10.2 (December 31, 2025)](#centreon-awie-25102)<br/>[25.10.1 (December 17, 2025)](#centreon-awie-25101)<br/>[25.10.0 (November 4, 2025)](#centreon-awie-25100) |
| **Centreon DSM** | [25.10.4 (June 29, 2026)](#centreon-dsm-25104)<br/>[25.10.3 (April 28, 2026)](#centreon-dsm-25103)<br/>[25.10.1 (December 17, 2025)](#centreon-dsm-25101)<br/>[25.10.0 (November 4, 2025)](#centreon-dsm-25100) |

</details>

## July 22, 2026

### Centreon Web 25.10.16

<details open>
  <summary>Enhancements</summary>

> - Added structured logging for all command/query bus dispatches and uncaught exceptions on the new kernel.

Reviewer: ```suggestion
- [Web] Added structured logging for all command/query bus dispatches and uncaught exceptions on the new kernel.
```

- [Administration] Centreon Web is now able to connect to the database using TLS 1.3.
- [Dashboards] Status grid widget: a new filter based on the resource state (unhandled, acknowledged, in downtime, flapping) has been added for the condensed view.
- [Resource status] Performance has been drastically improved for many search use cases.

> - [Upgrade] Upgrades are now fully traced in a dedicated upgrade.log file, making migrations easier to troubleshoot.

Reviewer: ```suggestion
- [Upgrade] A dedicated upgrade.log file now fully traces upgrades, making them easier to troubleshoot.
```

- [Web] A new logging system has been implemented for centreon-web.

</details>

<details>
  <summary>Bug fixes</summary>

- [API] Fixed an issue where duplicate host templates were silently accepted by the service template API.
- [Configuration] Fixed an issue where the '_' character was no longer valid in address definitions.
- [Configuration] Fixed stream connector (lua) parameters disappearing after deleting one of them in a Broker configuration.
- [Configuration] Fixed the incorrect Centreon Monitoring Agent install command for pollers that caused a 404 error.
- [Dashboards] It is now possible for users to share a dashboard with:
   * contact groups part of a common acl access group.
   * individual contacts listed in the contact groups they belong to.
- [Graphs] Non-admin users can now export metaservice performance graphs as CSV.
- [Installation] Fixed an issue where the installation wizard would not appear after a fresh install.
- [Resource status] Fixed an error where resources had a severity level above 100.

> - [Resource status] Fixed filtering with spaces around hyphens.

Reviewer: ```suggestion
- [Resource status]  Fixed an issue where filters did not return results when a host group name contained spaces around a hyphen.
```

- [Web] Fixed an issue causing SSO logout to fail due to an invalid format in SSO logout request.
- [Web] Fixed an issue in the extension manager where typing in the search bar or using type/status filters had no effect.

</details>

### Centreon Open Tickets 25.10.10

<details>
  <summary>Bug fixes</summary>

> - [Open tickets] Fixed “insufficient privileges” error when opening a ticket from a custom view without notification rules access.

Reviewer: ```suggestion
- [Open tickets] Fixed “insufficient privileges” error occurring when opening a ticket from a custom view without notification rules access.
```
